schemas/x509.yml

```diff
@@ -0,0 +1,154 @@
+---
+- name: x509
+  title: x509 Certificate
+  group: 2
+  description: These fields contain x509 certificate metadata.
+  type: group
+  reusable:
+    top_level: false
+    expected:
+      - file
+      - tls
+  fields:
+    - name: version_number
+      level: extended
+      type: long
+      description: Version of x509 format.
+      example: 3
+
+    - name: serial_number
+      level: extended
+      type: keyword
+      description: Unique serial number issued by the certificate authority.
+      example: 55:fb:b9:c7:de:bf:09:80:9d:12:cc:aa
+
+    - name: issuer.distinguished_name
+      level: extended
+      type: keyword
+      description: Distinguished name (DN) of issuing certificate authority.
+      example: C=US, O=DigiCert Inc, OU=www.digicert.com, CN=DigiCert SHA2 High Assurance Server CA
+
+    - name: issuer.common_name
+      level: extended
+      type: keyword
+      description: Common name (CN) of issuing certificate authority.
+      example: DigiCert SHA2 High Assurance Server CA
+
+    - name: issuer.organizational_unit
+      level: extended
+      type: keyword
+      description: List of organizational units (OU) of issuing certificate authority.
+      example: www.digicert.com
+
+    - name: issuer.organization
+      level: extended
+      type: keyword
+      description: Organizational units (O) of issuing certificate authority.
+      example: DigiCert Inc
+
+    - name: issuer.locality
+      level: extended
+      type: keyword
+      description: Locality name (L)
+      example: Mountain View
+
+    - name: issuer.state_or_province
+      level: extended
+      type: keyword
+      description: State or province name (ST, S, or P)
```

Comment:
Do you know if ST, S and P are mutually exclusive, in the distinguished names? If not we would have multiple source fields that map to one destination in ECS. This would be a problem

Reply:
they are all synonyms for the same OID in the ASN.1. It is tool dependent to how it is formatted. I think ST was probably the first standard developed in the United States (because we've historically been very inclusive :-p ), then later single letters of S for State or P for Province.